java/arcs/core/util/platform/native/PlatformInstant.kt

```diff
@@ -18,9 +18,11 @@ class PlatformInstant {
     fun toEpochMilli(): Long =
         TODO("Add support for ArcsInstant in Kotlin Native")
 
+    @Suppress("UNUSED_PARAMETER")
     fun compareTo(other: PlatformInstant): Int =
         TODO("Add support for ArcsInstant in Kotlin Native")
 
+    @Suppress("UNUSED_PARAMETER")
```

Comment:
This one seems weird. You are using the `other` parameter, so why is this needed?

Reply:
Ah, it's not! Removing it